src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -1062,6 +1063,13 @@ private Evaluable autoCoerceValue(final Type ltype, final Evaluable rhs, final B
       }
     }
 
+    if (ltype.equals(PATH_TYPE)) {
```

Comment:
@Veracity0 as the original author of the coercion code what do you think of this? Why do we only do this for typedefs and not for our own types? Should I extend this to include all of our enumerated types?